zadanie19_2.html

```diff
@@ -1,10 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Walidacja hasła</title>
+    <script>
+        function validatePassword() {
+            var password = document.getElementById("password").value;
+            var confirmPassword = document.getElementById("confirmPassword").value;
+
+            if (password.length < 8) {
+                alert("Hasło powinno zawierać minimum 8 znaków.");
+                return false;
+            }
+
+            if (password !== confirmPassword) {
+                alert("Hasło i powtórzone hasło nie są takie same.");
+                return false;
+            }
+            var passwordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]+$/;
```

Comment:
imponujący ten regex, szczerze mówiąc nie do końca rozumiem jak to działa, ale sprawdziłem i jest ok :)